modules/ROOT/pages/ch2-manage-props.adoc

```diff
@@ -102,31 +102,13 @@ CloudHub 2.0 reserves a set of properties and prevents apps from overwriting the
 If an application attempts to assign a value to a reserved property, CloudHub 2.0 doesn't use the assigned value.
 Instead, it returns the value as set internally, without throwing an error or exception, and adds a warning message to the log.
 
-Applications can reference reserved properties, for example, to retrieve the application name or deployment region.
-Use the `${csorganization.id}` and `${environment.id}` properties to retrieve those values if your app makes Anypoint Platform API calls.
-Use `${worker.id}` to identify each worker in a multiworker application.
-
-[IMPORTANT]
-Do not use the `${spring.profiles.active}` and `${logging.*}` properties.
-
 [%header,cols="3*a"]
 |===
 |Property|Description|Example
-|`${java.vendor}`|Vendor providing the JVM|Oracle Corporation
-|`${java.runtime.name}`|JVM name|Java&#8482; SE Runtime Environment
-|`${java.version}`|JVM version, excluding the dot (point) release.
-
-If a CloudHub 2.0 runtime release upgrades the JDK, this value changes.|1.8.0_141
-|`${domain}`|App name|`myapp`
 |`${http.port}`|HTTP port|8081
 |`${https.port}`|HTTPS port|8082
```

Comment:
it's always 8081 right?

Reply:
The four entries are for backward compatibility with CH1.0. Internally it would map to 8081 in CH2.0.